### PR TITLE
Add Spanish support

### PR DESCRIPTION
**Pull request by joebochill**
For PXBLUE-1950.

#### Changes proposed in this Pull Request:
- Configure namespaces to separate app-translations and workflow-translations
- Add documentation for how to support multiple languages in the application
- Add language toggle on Login screen of demo
- Add support for Spanish

### NOTE
This is a PR into a release branch for 2.2.0 since this change is not slated for release until after we implement the custom success screens in 2.1.0.

### CODE REVIEW

File: login-workflow/src/translations/i18n.ts [hunk 1]
```diff
@@ -7,12 +7,20 @@ void i18n
...
             fr_CA: translations.french,
+            fr_FR: translations.french,
+            // Spanish
+            es: translations.spanish,
+            es_US: translations.spanish,
+            es_MX: translations.spanish,
+            es_ES: translations.spanish,
         },
-        lng: 'en', // TODO get the device locale for the default
+        lng: 'es', // TODO get the device locale for the default
```
Comment (huayunh) [inline login-workflow/src/translations/i18n.ts:23]: does this mean we are using `es` by default

Comment (joebochill) [reply login-workflow/src/translations/i18n.ts:23]: Yeah, good catch. That's going to change to using the browser default in the commit that I'll be adding to this tomorrow.

**Comment by jeffvg** [comment]
should this be in correct language 

image

**Comment by jeffvg** [comment]
wondering if the 3 language toggles could be stuffed under debug mode

**Comment by joebochill** [comment]
> should this be in correct language

Good catch. There was an extra space in the key name which I have removed. 

> wondering if the 3 language toggles could be stuffed under debug mode

I was actually going to remove all of that (or at least comment it out) after the review. I left it in for your testing convenience.

File: login-workflow/docs/language-support.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# Language Support
+
+This package supports translations to different langauges using [i18next](https://www.i18next.com/) / [react-i18next](https://github.com/i18next/react-i18next). The workflow screens are currently available in:
```
Comment (jeffvg) [inline login-workflow/docs/language-support.md:3]: typo here - languages

Comment (joebochill) [reply login-workflow/docs/language-support.md]: Looks like I need to install my spellchecker plugin on the new mac.

**Review by jeffvg — APPROVED** [review]
looks good to me

**Pull request merged**